styles/arch-user-repository/catppuccin.user.css

```diff
@@ -0,0 +1,353 @@
+        #catppuccin(@lightFlavor, @accentColor);
+      }
+    }
+    @media (prefers-color-scheme: dark) {
+      :root {
+        #catppuccin(@darkFlavor, @accentColor);
+      }
+    }
+
+    :root[data-theme="dark"] {
+      #catppuccin(@darkFlavor, @accentColor);
+    }
+    :root[data-theme="light"] {
+      #catppuccin(@lightFlavor, @accentColor);
+    }
+
+    #catppuccin(@lookup, @accent) {
+      @rosewater: @catppuccin[@@lookup][@rosewater];
+      @flamingo: @catppuccin[@@lookup][@flamingo];
+      @pink: @catppuccin[@@lookup][@pink];
+      @mauve: @catppuccin[@@lookup][@mauve];
+      @red: @catppuccin[@@lookup][@red];
+      @maroon: @catppuccin[@@lookup][@maroon];
+      @peach: @catppuccin[@@lookup][@peach];
+      @yellow: @catppuccin[@@lookup][@yellow];
+      @green: @catppuccin[@@lookup][@green];
+      @teal: @catppuccin[@@lookup][@teal];
+      @sky: @catppuccin[@@lookup][@sky];
+      @sapphire: @catppuccin[@@lookup][@sapphire];
+      @blue: @catppuccin[@@lookup][@blue];
+      @lavender: @catppuccin[@@lookup][@lavender];
+      @text: @catppuccin[@@lookup][@text];
+      @subtext1: @catppuccin[@@lookup][@subtext1];
+      @subtext0: @catppuccin[@@lookup][@subtext0];
+      @overlay2: @catppuccin[@@lookup][@overlay2];
+      @overlay1: @catppuccin[@@lookup][@overlay1];
+      @overlay0: @catppuccin[@@lookup][@overlay0];
+      @surface2: @catppuccin[@@lookup][@surface2];
+      @surface1: @catppuccin[@@lookup][@surface1];
+      @surface0: @catppuccin[@@lookup][@surface0];
+      @base: @catppuccin[@@lookup][@base];
+      @mantle: @catppuccin[@@lookup][@mantle];
+      @crust: @catppuccin[@@lookup][@crust];
+      @accent-color: @catppuccin[@@lookup][@@accent];
+
+      color-scheme: if(@lookup = latte, light, dark);
+
+      ::selection {
+        background-color: fade(@accent-color, 30%);
+      }
+
+      input,
+      textarea {
+        &::placeholder {
+          color: @subtext0 !important;
+        }
+      }
+
+      /* Start styling your website here: */
+      background-color: @base;
+      body {
+            background-color: @base;
+            color: @text
+        }
+        div.box {
+            background-color: @surface1;
+            border-color: @accent-color
+        }
+
+
+  // archnavbar
+        #archnavbar {
+            background-color: @base !important;
+            border-bottom-color: @accent-color !important;
+            a {
+                color: @text !important
+            }
+            a:hover {
+                color: @accent-color !important
+            }
+        }
+
+  // archdev-navbar
+        #archdev-navbar li a {
+            color: @accent-color !important
+        }
+        #archdev-navbar li button {
+            color: @accent-color !important
+        }
+
+
+  // idk
+        content {
+            color: @accent-color
+            }
+
+
+  // Links
+        a {
+            color: @accent-color !important;
+            text-decoration: none !important;
```

Comment:
why are we doing this?

Reply:
idk but it works

Reply:
We shouldn't change non-color things, like text-decoration. Why is the change needed?